collect_app/src/androidTest/java/org/odk/collect/android/support/CollectHelpers.kt

```diff
@@ -16,4 +16,12 @@ object CollectHelpers {
         application.component = testComponent
         return testComponent
     }
+
+    fun simulateProcessRestart(appDependencyModule: AppDependencyModule? = null) {
```

Comment:
Any static state will not be reset here - just another reason to avoid that at all costs! I feel like this is as close as I could reasonably get.